### PR TITLE
Regression: Fix size of custom emoji and render emoji on thread message preview

### PR DESCRIPTION
**Pull request by filipemarins**
## Issue(s)

Fix: emoji name display `:undefined:` in message template
Fix: size of custom emoji when is `BigEmoji`
Fix: size of message reaction when is custom emoji
Fix: the thread preview was not parsing the message

## Further comments

The custom emoji was not rendering with the correct size. And also add the message render to thread preview message.
Fix the message reaction size of custom emoji.

### CODE REVIEW

File: apps/meteor/client/components/Message/MessageBodyRender/BigEmoji.tsx [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
-		))}
-	</>
-);
+const BigEmoji: FC<BigEmojiProps> = ({ value, disableBigEmoji }) => {
+	const bigEmojiClass = css`
+		> span {
+			width: ${disableBigEmoji ? '1em' : '44px'};
+			height: ${disableBigEmoji ? '1em' : '44px'};
+		}
+	`;
```
Comment (ggazzo) [inline apps/meteor/client/components/Message/MessageBodyRender/BigEmoji.tsx:18]: do not always use styling on application side, please adjust component on fuselage side

**Review by ggazzo — CHANGES_REQUESTED** [review]

**Review by ggazzo — DISMISSED** [review]
consider create a MessageBodyRender for Threads like ThreadBodyRender instead of polluting the components

File: apps/meteor/client/components/Message/MessageBodyRender/BigEmoji.tsx (hunk 1 above)
Comment (filipemarins) [reply apps/meteor/client/components/Message/MessageBodyRender/BigEmoji.tsx:18]: This isn't a fuselage component, I am styling the component rendered by the emoji parser... I've fixed the stylesheet used by the parse, but @gabriellsh asked to not rely on external CSS and create the style on react component.

**Comment by filipemarins** [comment]
> consider create a MessageBodyRender for Threads like ThreadBodyRender instead of polluting the components

I've created the `MessageRender` component, the problem with the `ThreadBody` name is that can cause some mistakes with thread message rendering, instead of thread preview.

Also, the `MessageRender` was reused on message content.

**Review by gabriellsh — APPROVED** [review]

**Pull request merged**